components/build-service/production/stone-prod-p02/webhook-config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "https://github.com": "https://smee-smee.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/redhathook12",
+    "https://gitlab.com": "https://smee-smee.apps.stone-prd-host1.wdlc.p1.openshiftapps.com/redhathook12"
```

Comment:
This is fine. it should be on the public host since it recives the wehooks from GitHub. The thing is that it should use a different channel (different url) so it won't consume the events that p1 needs to consume.

Reply:
I'll configure the new cluster to use another channel.